Fix to use scalar terraform examples (#99)
scalar-labs/scalar-kubernetes: docs/AKSScalarTerraformDeploymentGuide.md

This document shows how to create Azure Kubernetes Service (AKS) along with a virtual network, a bastion host, and database resources using Terraform scripts.

> The Terraform scripts for creating resources with [scalar-terraform](https://github.com/scalar-labs/scalar-terraform) are all stored in [scalar-terraform-examples](https://github.com/scalar-labs/scalar-terraform-examples).

Reviewer: `stored` sounds somewhat unnatural to me because the examples have been moved.

```suggestion
The Terraform scripts for creating resources with [scalar-terraform](https://github.com/scalar-labs/scalar-terraform) are available in a separate repository: [scalar-terraform-examples](https://github.com/scalar-labs/scalar-terraform-examples).
```

Author: Thanks! Fixed in cb21870.

* 1 Bastion instance with a public IP
* 1 Monitor instance to collect metrics and logs

## Clone scalar-teraform-examples

```console

> $ git clone https://github.com/scalar-labs/scalar-terraform-examples -b <master or tag> modules

Reviewer: I don't think it is the best way to clone another repository inside the repository. There would be no problem with cloning the examples outside of the `${SCALAR_K8S_HOME}`. How about introducing a new environment variable, `SCALAR_TERRAFORM_EXAMPLES`?

Author: Good point! 👍 

Fixed in 0999727
